.github/workflows/ivts.yaml

```diff
@@ -0,0 +1,92 @@
+#
```

Comment:
I think we could do with a README.md explaining what this module is, does and why it's here, and how it gets used ?

Reply:
Sure, I will write a README when the POC works and I'm ready to deliver this PR.